wye/workshops/templatetags/workshop_action_button.py

```diff
@@ -8,20 +8,32 @@ def show_draft_button(workshop, user):
     if workshop.status in [WorkshopStatus.REQUESTED,
                            WorkshopStatus.ACCEPTED,
                            WorkshopStatus.DECLINED]:
-        return True
+
+        requester_members = workshop.requester.user.values()
+
+        for member in requester_members:
```

Comment:
In stead of loop you can just do this
if user in w.requester.user.all():  # list of organization users
    return True